plugins/dist_table_procedure.h

```diff
@@ -185,10 +185,12 @@ struct DistTableProcedure: public Procedure {
 
     DistTableProcedureConfig procConfig;
 
-    static void persist(const Url & modelFileUrl, DistTableMode mode, 
+    static void persist(const Url & modelFileUrl, DistTableMode mode,
                         const DistTablesMap & distTablesMap);
 
-    static const int DIST_TABLE_PERSIST_VERSION = 2;
+    static constexpr int DIST_TABLE_PERSIST_VERSION() {
+        return 2;
+    }
```

Comment:
this can be fixed with constexpr.  You need to actually define static variables once in the .cc file.